### PR TITLE
add suite of FASTA tools to the IUC repository

### PR DESCRIPTION
**Pull request by bgruening**
I would like to create some basic tools suites and collect a bunch of high quality tools that we as IUC can recommend. 

The first one is: Tools to manipulate FASTA files

The following suites I would like to add if we agree this is a good idea for the IUC.
* GFF/GTF tools
* Sequence Searches (BLAST, Diamond, Kraken etc...)
* Exome-Seq (all different callers + Annotation)
* Statistics

### CODE REVIEW

File: suites/suite_fasta_tools/repository_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<repositories description="This is a metapackage that will install all vcftools wrappers and dependencies.">
```
Comment (jmchilton) [inline suites/suite_fasta_tools/repository_dependencies.xml]: Copy and paste problem with description?

File: suites/suite_fasta_tools/repository_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <repository name="fasta_filter_by_length" owner="devteam" />
+    <repository name="seq_filter_by_id" owner="peterjc" />
+    <repository name="seq_select_by_id" owner="peterjc" />
+    <repository name="seq_rename" owner="peterjc" />
+    <repository name="fasta_formatter" owner="devteam" />
+    <repository name="fasta_to_tabular" owner="devteam" />
+    <repository name="tabular_to_fasta" owner="devteam" />
+    <repository name="fasta_nucleotide_changer" owner="devteam" />
+    <repository name="find_subsequences" owner="bgruening" />
+    <repository name="suite_fastx_toolkit_0_0_13" owner="devteam" />
```
Comment (erasche) [inline suites/suite_fasta_tools/repository_dependencies.xml]: If you're adding fastx toolkit, you might add seqtk as well. Does fasta/q

**Comment by erasche** [comment]
I like the idea...we should do more of this. Making suites that admins want to install that provides lots of useful functionality/tools.

I'd +1 an "assembly" suite as well, since that's pretty common.

File: suites/suite_fasta_tools/repository_dependencies.xml (hunk 1 above)
Comment (bgruening) [inline suites/suite_fasta_tools/repository_dependencies.xml]: Fixed, thanks!

**Comment by bgruening** [comment]
@erasche added seqtk, thanks!

**Comment by peterjc** [comment]
This looks like a good idea :)

**Pull request merged**